app/assets/stylesheets/page.sass

```diff
@@ -14,7 +14,7 @@ body
   padding: 10px 10px 0 10px
 
 header
-  background-color: #242021
+  background-color: white
```

Comment:
The color keywords is so much clearer!  I don't see white as one of our defined variables.  Is white always the same white across all browsers?  I see we are replacing different colors with white . (see line 83 below) . Is that intentional?

Reply:
We decided to use `white` as the background, and perhaps it is a better practice to define something like `$white: #ffffff` rather than rely on the CSS name iteself.